app/scripts/controller-init/notifications/notification-services-push-controller-init.ts

```diff
@@ -109,7 +119,8 @@ export const NotificationServicesPushControllerInit: ControllerInitFunction<
           /* eslint-disable @typescript-eslint/naming-convention */
           notification_id: notification.id,
           notification_type: notification.type,
-          chain_id: chainId,
+          ...otherNotificationProperties(),
+          data: notification, // data blob for feature teams to analyse their notification shapes
```

Comment:
Aligning with mobile analytics, adding a data "blob" for product/feature teams to perform any advanced mixpanel segmentation.